custom-namespaces.yaml

```
@@ -732,6 +739,10 @@ firefox_accounts:
       type: table_explore
       views:
         base_view: health_issues
+    health_issues:
```

There's already a `health_issues` explore defined above.
```suggestion
    health_lighthouse:
```

---

fixed in new push. thanks